application/client/src/app/ui/elements/timezones/timezone.ts

```diff
@@ -30,38 +34,23 @@ export class Timezone {
         );
     }
 
-    public getNameAsHtml(): string {
-        if (this._filter === '') {
-            return this.name;
-        }
-        const reg = regex.fromStr(this._filter);
-        if (reg instanceof Error) {
-            return this.name;
-        }
-        return this.name.replace(reg, (match): string => {
-            return `<span>${match}</span>`;
-        });
+    public get html_name(): string {
+        return this._html_name;
```

Comment:
In JS name convention is `htmlName` instead `html_name`
We have probably one exception - when we have deal with parsing income from rust-world JSON... but at places where we can keep JS naming convention, we should keep it